app/assets/javascripts/app/views/comment_stream_view.js

```diff
@@ -21,12 +23,20 @@ app.views.CommentStream = app.views.Base.extend({
   setupBindings: function() {
     this.model.comments.bind("add", this.appendComment, this);
     this.model.comments.bind("remove", this.removeComment, this);
+    $(document.body).click(this.onFormBlur.bind(this));
   },
 
   postRenderTemplate : function() {
     this.model.comments.each(this.appendComment, this);
     this.commentBox = this.$(".comment_box");
     this.commentSubmitButton = this.$("input[name='commit']");
+
+    this.mdEditor = new Diaspora.MarkdownEditor(this.$(".comment_box"), {
+      onPreview: function($mdInstance) {
+        return "<div class='preview-content'>" + app.helpers.textFormatter($mdInstance.getContent()) + "</div>";
+      },
+      onFocus: this.openForm.bind(this)
+    });
```

Comment:
Please add a test that we set `mdEditor` and create a new `Diaspora.MarkdownEditor`.